src/test/java/school/cesar/eta/unit/PeopleComparatorTest.java

```diff
@@ -0,0 +1,254 @@
+package school.cesar.eta.unit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import java.time.LocalDate;
+
+import org.mockito.junit.jupiter.MockitoExtension;
+import school.cesar.eta.unit.entity.Person;
+import static org.mockito.Mockito.when;
+import java.time.LocalDate;
+import java.time.Year;
+
+@ExtendWith(MockitoExtension.class)
+public class  PeopleComparatorTest {
+    @InjectMocks
+    PeopleComparator application;
```

Comment:
É necessário se ter controle sobre a datas na execução dos casos de teste.

Isso pode ser feito com mokito:
```java
when(application.getCurrentDate()).thenReturn(LocalDate.of(1980,05,29));
```
ou quando a classe `PeopleComparator` for instanciada:
```java
application = new PeopleComparator () {
    @override
    public LocalDate getCurrentDate() {
        return LocalDate.of(1980,05,29);
    }
};
```